src/browser/browser.ts

```diff
@@ -213,19 +217,23 @@ export class Browser {
 
     for (let i = 0; i < scrolls; i++) {
       await page.evaluate(
-        (scrollByHeight, scrollDivSelector) => {
-          document.querySelector(scrollDivSelector)?.scrollBy(0, scrollByHeight);
+        (scrollByHeight, scrollElementSelector) => {
+          scrollElementSelector === 'body'
+            ? window.scrollBy(0, scrollByHeight)
+            : document.querySelector(scrollElementSelector)?.scrollBy(0, scrollByHeight);
         },
         heights.dashboard.client,
-        scrollDivSelector
+        scrollElementSelector,
```

Comment:
This code now has more logic, and it is the same as below. Can we extract this and call it with height 0 in the latter case?

Reply:
one uses `scrollBy` and the other uses `scrollTo`

i guess we could change the first use to `scrollTo` and multiply the loop index, but i was trying to avoid modifying the existing code tbh